app/consumers/concerns/report_parsing.rb

```diff
@@ -34,16 +34,15 @@ def enqueue_parse_report_job(reports)
     def notify_report_success(profile_id, job)
       msg = "Enqueued report parsing of #{profile_id}"
       msg += " from request #{request_id} as job #{job}"
-      logger.info(msg)
```

Comment:
Is the INFO log removed due to duplicity?

Reply:
Yes